### DISCUSSION TITLE
Mass flow rate measurment

### DISCUSSION THREAD

**Discussion by HunterXHunter1997** [firemodels/fds#13020] [Q&A]
Hello team,
I am testing a nozzle. my goal is to reach the same actual water pattern and the same volume flow rate below the nozzle.

So I added a set of devices below my nozzle to measure the volume flow rate as follows.
&DEVC ID='FLOW1', QUANTITY='VOLUME FLOW -', XB=2.0,2.2,2.0,2.2,1.0,1.0/
&DEVC ID='FLOW01', QUANTITY='VOLUME FLOW', XB=2.2,2.4,2.0,2.2,1.0,1.0/
&DEVC ID='FLOW02', QUANTITY='VOLUME FLOW', XB=2.4,2.6,2.0,2.2,1.0,1.0/
&DEVC ID='FLOW03', QUANTITY='VOLUME FLOW', XB=2.6,2.8,2.0,2.2,1.0,1.0/
&DEVC ID='FLOW04', QUANTITY='VOLUME FLOW', XB=2.8,3.0,2.0,2.2,1.0,1.0/
&DEVC ID='FLOW05', QUANTITY='VOLUME FLOW', XB=3.0,3.2,2.0,2.2,1.0,1.0/
&DEVC ID='FLOW06', QUANTITY='VOLUME FLOW', XB=2.0,2.2,2.2,2.4,1.0,1.0/
&DEVC ID='FLOW07', QUANTITY='VOLUME FLOW', XB=2.2,2.4,2.2,2.4,1.0,1.0/
&DEVC ID='FLOW08', QUANTITY='VOLUME FLOW', XB=2.4,2.6,2.2,2.4,1.0,1.0/
&DEVC ID='FLOW09', QUANTITY='VOLUME FLOW', XB=2.6,2.8,2.2,2.4,1.0,1.0/
&DEVC ID='FLOW10', QUANTITY='VOLUME FLOW', XB=2.8,3.0,2.2,2.4,1.0,1.0/
&DEVC ID='FLOW11', QUANTITY='VOLUME FLOW', XB=3.0,3.2,2.2,2.4,1.0,1.0/
&DEVC ID='FLOW12', QUANTITY='VOLUME FLOW', XB=2.0,2.2,2.4,2.6,1.0,1.0/
&DEVC ID='FLOW13', QUANTITY='VOLUME FLOW', XB=2.2,2.4,2.4,2.6,1.0,1.0/
&DEVC ID='FLOW14', QUANTITY='VOLUME FLOW -', XB=2.4,2.6,2.4,2.6,1.0,1.0/
&DEVC ID='FLOW15', QUANTITY='VOLUME FLOW', XB=2.6,2.8,2.4,2.6,1.0,1.0/
&DEVC ID='FLOW16', QUANTITY='VOLUME FLOW', XB=2.8,3.0,2.4,2.6,1.0,1.0/
&DEVC ID='FLOW17', QUANTITY='VOLUME FLOW', XB=3.0,3.2,2.4,2.6,1.0,1.0/
&DEVC ID='FLOW18', QUANTITY='VOLUME FLOW', XB=2.0,2.2,2.6,2.8,1.0,1.0/
&DEVC ID='FLOW19', QUANTITY='VOLUME FLOW', XB=2.2,2.4,2.6,2.8,1.0,1.0/
&DEVC ID='FLOW20', QUANTITY='VOLUME FLOW', XB=2.4,2.6,2.6,2.8,1.0,1.0/
&DEVC ID='FLOW21', QUANTITY='VOLUME FLOW', XB=2.6,2.8,2.6,2.8,1.0,1.0/
&DEVC ID='FLOW22', QUANTITY='VOLUME FLOW', XB=2.8,3.0,2.6,2.8,1.0,1.0/
&DEVC ID='FLOW23', QUANTITY='VOLUME FLOW', XB=3.0,3.2,2.6,2.8,1.0,1.0/
&DEVC ID='FLOW24', QUANTITY='VOLUME FLOW', XB=2.0,2.2,2.8,3.0,1.0,1.0/
&DEVC ID='FLOW25', QUANTITY='VOLUME FLOW', XB=2.2,2.4,2.8,3.0,1.0,1.0/
&DEVC ID='FLOW26', QUANTITY='VOLUME FLOW', XB=2.4,2.6,2.8,3.0,1.0,1.0/
&DEVC ID='FLOW27', QUANTITY='VOLUME FLOW', XB=2.6,2.8,2.8,3.0,1.0,1.0/
&DEVC ID='FLOW28', QUANTITY='VOLUME FLOW', XB=2.8,3.0,2.8,3.0,1.0,1.0/
&DEVC ID='FLOW29', QUANTITY='VOLUME FLOW', XB=3.0,3.2,2.8,3.0,1.0,1.0/
&DEVC ID='FLOW30', QUANTITY='VOLUME FLOW', XB=2.0,2.2,3.0,3.2,1.0,1.0/
&DEVC ID='FLOW31', QUANTITY='VOLUME FLOW', XB=2.2,2.4,3.0,3.2,1.0,1.0/
&DEVC ID='FLOW32', QUANTITY='VOLUME FLOW', XB=2.4,2.6,3.0,3.2,1.0,1.0/
&DEVC ID='FLOW33', QUANTITY='VOLUME FLOW', XB=2.6,2.8,3.0,3.2,1.0,1.0/
&DEVC ID='FLOW34', QUANTITY='VOLUME FLOW', XB=2.8,3.0,3.0,3.2,1.0,1.0/
&DEVC ID='FLOW35', QUANTITY='VOLUME FLOW', XB=3.0,3.2,3.0,3.2,1.0,1.0/
 
My question is, Do these devices measure the volume flow rate for the air or the water species that comes from the nozzle? because the charts that I had were too far from the actual values.
 Thanks a lot.

**Comment by mcgratta** [comment]
These lines will measure the volume flow of gas through a plane defined by XB.

**Comment by HunterXHunter1997** [comment]
Thank for reply !
So is there any way to measure the mass/volume flow rate of the water? just to be sure that the nozzle injects the correct quantity of water

**Reply by drjfloyd** [reply]
22.9 Particle Output Quantities

**Comment by Er9y714** [comment]
For information, VOLUME FLOW quantity is deprecated in the latest versions if I am not mistaken. I would recommend running the latest FDS version.
